src/bionics.cpp

```diff
@@ -204,21 +204,18 @@ bionic_data const& bionic_info(std::string const &id)
     debugmsg("bad bionic id");
 
     static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0,
-                                         "bad_bionic", false,
-                                         { num_bp , 1 } };
+                                         "bad_bionic", false };
     return null_value;
 }
 
 void bionics_install_failure( player *u, int difficulty, int success,
-                              std::map<body_part, int> affected_bodyparts );
+                              std::map<body_part, size_t> affected_bodyparts );
 
 bionic_data::bionic_data(std::string nname, bool ps, bool tog, int pac, int pad, int pot,
-                         int ct, int cap, std::string desc, bool fault,
-                         std::map<body_part, int> aff_bps
+                         int ct, int cap, std::string desc, bool fault
 ) : name(std::move(nname)), description(std::move(desc)), power_activate(pac),
```

Comment:
`std::move` is probably a no-op here. If it isn't there's a related problem as you haven't given the caller any indication you intend to steal their string.

You could try `const std::string& desc` as a parameter and then also specify `const std::string description` in the header presuming there's no reason to change the description.